_authors/felipe-bohorquez-perry.md

```diff
@@ -6,4 +6,6 @@ full_name: Felipe Bohorquez-Perry
 role: Scrum Master
 image: "/assets/img/team/felipe-square.jpg"
 active: true
+thinking: Where is everything?
+doing: AGILE. Design thinking. Working with devs. Scrum, Film, VFX, software, learning.
```

Comment:
Agile, Scrum, pragmatism, value with values, better ways of working, new tools, old tools, human behaviours, incentivizing our best impulses, Music, VFX, films and how it all connects.